crates/hdk/src/guest_callback/entry_defs.rs

```diff
@@ -66,7 +66,7 @@ macro_rules! entry_def {
             fn try_from(entry: &$crate::prelude::Entry) -> Result<Self, Self::Error> {
                 match entry {
                     $crate::prelude::Entry::App(eb) => Ok(Self::try_from(
-                        $crate::prelude::SerializedBytes::from(eb.to_owned()),
+                        $crate::prelude::SerializedBytes::try_from(eb.to_owned())?,
```

Comment:
Was this a change due to a newer version of serialized bytes?
I don't think you need the `to_owned()`?